cpp/src/parquet/stream_reader.h

```diff
@@ -41,10 +42,17 @@ namespace parquet {
 /// The user must explicitly advance to the next row using the
 /// EndRow() function or EndRow input manipulator.
 ///
-/// Currently there is no support for optional or repeated fields.
+/// Required and optional fields are supported:
+/// - Required fields are read using operator>>(T)
+/// - Optional fields are read with
+///   operator>>(arrow::util::optional<T>)
+/// Currently there is no support for repeated fields.
 ///
 class PARQUET_EXPORT StreamReader {
  public:
+  template <typename T>
+  using optional = arrow::util::optional<T>;
```

Comment:
sorry I thought I commented (and perhaps I missed the response) but I'm kind of against this alias.

Reply:
On the contrary this alias is useful to users of this class so that they can use the required optional type.